tests/installation/addon_products_via_SCC_yast2.pm

```diff
@@ -15,29 +15,51 @@ use base qw(y2logsstep y2x11test);
 use strict;
 use testapi;
 use registration 'fill_in_registration_data';
+use version_utils 'sle_version_at_least';
 
-sub run {
-    my ($self) = @_;
-    x11_start_program('xterm');
+sub prepare_reg_sle12 {
```

Comment:
how about having single function prepare_reg which check version inside ?

Reply:
I've wrapped it in ```test_setup``` already, so don't really want to put it in one piece of code.